baseapp/datadog/datadog.go

```diff
@@ -38,10 +38,9 @@ import (
 	"time"
 
 	"github.com/DataDog/datadog-go/statsd"
+	"github.com/palantir/go-baseapp/baseapp"
 	"github.com/pkg/errors"
 	"github.com/rcrowley/go-metrics"
-
-	"github.com/palantir/go-baseapp/baseapp"
```

Comment:
Still not a fan of this dependency reordering but I'm not sure if it's worth fixing...